File: 0531-lonely-pixel-i/0531-lonely-pixel-i.py
class Solution:
    def findLonelyPixel(self, p: List[List[str]]) -> int:
        n, m = len(p), len(p[0])
        rows = [0 for _ in range(n)]
        cols = [0 for _ in range(m)]

        for i in range(n):
            for j in range(m):
                if p[i][j] == 'B':
                    rows[i] += 1
                    cols[j] += 1
        
        return sum(1 for i in range(n) for j in range(m) if p[i][j] == 'B' and rows[i] == 1 and cols[j] == 1)

File: 0531-lonely-pixel-i/0531-lonely-pixel-i.ts
function findLonelyPixel(p: string[][]): number {
    let n: number = p.length, m: number = p[0].length;
    const rows = new Array(n).fill(0);
    const cols = new Array(m).fill(0);

    for (let i: number = 0; i < n; ++i) {
        for (let j: number = 0; j < m; ++j) {
            if (p[i][j] == 'B') {
                rows[i]++;
                cols[j]++;
            }
        }
    }

    let c: number = 0;
    for (let i: number = 0; i < n; i++) {
        for (let j: number = 0; j < m; ++j) {
            if (p[i][j] == 'B' && rows[i] == 1 && cols[j] == 1)
                c++;
        }
    }
    return c;
};